aqua/install_pkc.sh

```diff
@@ -73,9 +73,9 @@ while ! install_media_wiki; do
         echo "MediaWiki intallation retries exceeded"
         break
     fi
-    echo "Retrying MediaWiki installation"
-    ((retry_counter++))
+    retry_counter=$((retry_counter+1))
     sleep 5
+    echo "Retrying MediaWiki installation"
```

Comment:
This should be before the `sleep 5`. Otherwise, people have no idea why it is sleeping for 5 s.

Reply:
ok yup, wanted it to print just before the install 👍